<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>QgsComposerTextWidgetBase</class>
 <widget class="QWidget" name="QgsComposerTextWidgetBase">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>396</width>
    <height>322</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_2">
   <item row="0" column="0">
    <widget class="QToolBox" name="mToolBox">
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="mTextToolboxPage">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>378</width>
        <height>277</height>
       </rect>
      </property>
      <attribute name="label">
       <string>Text</string>
      </attribute>
      <layout class="QGridLayout" name="gridLayout">
       <item row="0" column="0">
        <layout class="QHBoxLayout" name="horizontalLayout_3">
         <item>
          <widget class="QLabel" name="mResizeModeLabel">
           <property name="text">
            <string>Resize mode</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QComboBox" name="mResizeModeComboBox"/>
         </item>
        </layout>
       </item>
       <item row="1" column="0">
        <layout class="QHBoxLayout" name="horizontalLayout">
         <item>
          <widget class="QFontComboBox" name="mFontComboBox"/>
         </item>
         <item>
          <spacer name="horizontalSpacer">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>38</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="QSpinBox" name="mFontSizeSpinBox">
           <property name="minimum">
            <number>1</number>
           </property>
           <property name="value">
            <number>10</number>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="mBoldPushButton">
           <property name="text">
            <string>B</string>
           </property>
           <property name="checkable">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="mItalicsPushButton">
           <property name="text">
            <string>I</string>
           </property>
           <property name="checkable">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QgsColorButton" name="mFontColorButton">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="text">
            <string/>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item row="2" column="0">
        <widget class="QTextEdit" name="mTextEdit"/>
       </item>
       <item row="3" column="0">
        <layout class="QHBoxLayout" name="horizontalLayout_2">
         <item>
          <widget class="QLabel" name="mBackgroundColorLabel">
           <property name="text">
            <string>Background color</string>
           </property>
           <property name="buddy">
            <cstring>mBackgroundColorButton</cstring>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QgsColorButton" name="mBackgroundColorButton">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="text">
            <string/>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QgsColorButton</class>
   <extends>QToolButton</extends>
   <header>qgscolorbutton.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
